tests/by-util/test_arch.rs

```diff
@@ -21,3 +21,10 @@ fn test_arch_help() {
 fn test_invalid_arg() {
     new_ucmd!().arg("--definitely-invalid").fails_with_code(1);
 }
+
+#[test]
+fn test_arch_output_is_not_empty() {
+    let result = new_ucmd!().run();
+    result.success();
```

Comment:
You can simplify the test a bit by using `succeeds()`:
```suggestion
    let result = new_ucmd!().succeeds();
```